package.json

```diff
@@ -21,12 +21,12 @@
     "breakjs": "^1.0.0",
     "classnames": "^2.2.5",
     "css-element-queries": "^1.0.1",
-    "patternfly": "^3.38.0",
-    "react-bootstrap": "^0.31.5",
+    "patternfly": "^3.42.0",
+    "react-bootstrap": "^0.32.1",
```

Comment:
we definitely want to check for regressions on this one... amongst the others ;)

Reply:
@priley86 added storybook in first post 😋